src/css/bundle.scss

```diff
@@ -1014,4 +1138,400 @@ input[type='text'].display {
+    color: $red;
+}
+
+body .quest-tasks ul {
+    background: #00000047;
+}
+
+body .window-app.forien-quest-preview .quest-body .quest-info .quest-col-right header {
+    border-block-end: 2px solid var(--palette-primary,var(--default-primary-accent,#782e22));
+    border-bottom: 2px solid var(--palette-primary,var(--default-primary-accent,#782e22));
+    color: $edge;
+    display: flex;
+    flex: 0 0 1px;
+    margin-bottom: 4px;
+    background: #000000b8;
+}
+//table h2
+body #forien-quest-log .quest-log .table ul li.drag-quest {
+    align-items: center;
+    background: #000000b8;
+    border: 1px solid #0000;
+    border-radius: 5px;
+    display: flex;
+    justify-content: flex-start;
+    margin: 0 4px 2px 0;
+    min-height: 42px;
+    transition: border-color .3s ease,box-shadow .3s ease;
+}
+//title h2
+body #forien-quest-log .quest-log .table ul .title h2 {
+    color: #C7B002;
+    background: #00000000;
+}
+
+body .open-quest.tasks {
+    color: #f2f2f2; /* Sets the background color */
+}
+
+//Buttons
+body .window-app.forien-quest-preview .quest-body .quest-info .quest-col-right button {
+    flex: 0 0 1px;
+    font-size: 15px;
+    height: 20px;
+    line-height: 1;
+    white-space: nowrap;
+    background: $darkred;
+    color: white;
+}
+
+#forien-quest-log button, .window-app.forien-quest-preview button {
+    border-radius: 5px;
+    cursor: pointer;
+    height: 30px;
+    margin-bottom: 2px;
+    margin-right: 4px;
+    margin-top: 2px;
+    transition: border-color .3s ease,background .3s ease,box-shadow .3s ease;
+    background: $darkred;
+    color: white;
+}
+//Play
+#forien-quest-log .actions i.move.fas.fa-play:hover, .window-app.forien-quest-preview .actions i.move.fas.fa-play:hover {
+    color: green;
+}
+//Trashbin
+body #forien-quest-log .actions i.delete.fas.fa-trash:hover, body .window-app.forien-quest-preview .actions i.delete.fas.fa-trash:hover {
+    color: #cc5e5e;
+}
+
+body #forien-quest-log .actions i.fa-trash, body .window-app.forien-quest-preview .actions i.fa-trash {
+    color: #af0505;
+}
+//Back to deactive
+body #forien-quest-log .actions i.move.fas.fa-stop-circle:hover, body .window-app.forien-quest-preview .actions i.move.fas.fa-stop-circle:hover {
+    color: $edge;
+}
+
+//Make Available
+#forien-quest-log .actions i.move.fas.fa-clipboard:hover, .window-app.forien-quest-preview .actions i.move.fas.fa-clipboard:hover {
+    color: $edge;
+}
+
+body #forien-quest-log .quest-log .table ul .title p {
+    color: white;
+}
+//End Quest Log Styles
+
+.list-header.item-section[data-item-id="weapon"][data-item-type=""] {
+    color: $edge;  /* set the color to yellow */
+}
+
+.common-test{
+margin-bottom: 4px; margin-top: 4px;
+color: white;
+}
+
+.custom-item-element {
+    color: white;               /* Text color */
+}
+//lable back to white color
+.matrix-label{
+    color: white;
+}
+
+.scroll-area-matrix{
+    overflow-y: auto;
+    color: white;
+}
```

Comment:
Can these styles up till quest log exist in .sr5 scope?
That we can keep global css scope clean and limit unwanted sideeffects.

EDIT: This goes for all styles from here on out.